commands/host.vesnin

```diff
@@ -0,0 +1 @@
+host.nicole
```

Comment:
???
If it is a dummy file, just leave it empty. Or just remove it from git.

Reply:
It is not a dummy file. It will be installed on vesnin machines (if we wish). We need to split host-specific handlers.

Reply:
As far as I can see, you do machine switching inside the `host.nicole` script.
I would understand if you had a `host` script that would source a `host.$MACHINE` script with machine-specific commands and then would just call machine specific callbacks like `nmi_machine`, but doing direct machine switching in `host.nicole` and then having this dummy doesn't look neat.

I'd even expect `clicmd` to pre-load the machine-specific command file before running the main handler that could then check for availability of a machine-specific function (using `type`) and behave accordingly (e.g. report that the function is not supported for the machine).